MAUI/DataGrid/export-to-excel.md

```diff
@@ -0,0 +1,548 @@
+
+{% tabs %}
+{% highlight c# %}
+DataGridPdfExportingOption option = new DataGridPdfExportingOption();
+        var list = new List<string>();
+        list.Add("EmployeeID");
+        list.Add("Name");
+        option.ExcludedColumns = list;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Exclude columns when exporting" src="Images\Export-to-Excel\maui-datagrid-excluded-columns.png" width="689"/>
+
+### Export Selected Rows of SfDataGrid
+
+SfDataGrid allows you to export only the currently selected rows in the grid to the worksheet using the `DataGridExcelExportingController.ExportToExcel` method by passing the instance of the SfDataGrid and `SfDataGrid.SelectedRows` collection as an argument.
+
+Refer the below code to export the selected rows alone to Excel.
+
+{% tabs %}
+{% highlight c# %}
+ObservableCollection<object> selectedItems = dataGrid.SelectedRows;
+var excelEngine = excelExport.ExportToExcel(this.dataGrid, selectedItems);
+{% endhighlight %}
+{% endtabs %}
+
+
+### Customize Exporting Excel Version
+
+The SfDataGrid allows exporting the data to Excel in specific versions by using the `DataGridExcelExportingOption.ExcelVersion` property.
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+option.ExcelVersion = Syncfusion.XlsIO.ExcelVersion.Excel2013;
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting the grid from a specified row and column index 
+
+#### StartColumnIndex
+
+By default, the exported SfDataGrid will start from the 0th column in the Excel sheet. You can specify the starting column by using the `DataGridExcelExportingOption.StartColumnIndex` property.
+
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+option.StartColumnIndex = 3;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Export DataGrid to Excel format from the specified column index" src="Images\Export-to-Excel\maui-datagrid-start-column-index.png" width="689"/>
+
+#### StartRowIndex
+
+By default, the exported SfDataGrid will start from the 0th row in the Excel sheet. You can specify the starting row by using the `DataGridExcelExportingOption.StartRowIndex` property.
+{% tabs %}
+{% highlight c# %}
+DataGridExcelExportingOption option = new DataGridExcelExportingOption();
+option.StartRowIndex = 3;
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Export DataGrid to Excel format from the specified row index" src="Images\Export-to-Excel\maui-datagrid-start-row-index.png" width="689"/>
+
+### Exporting with sorting and filtering
+
+The SfDataGrid allows exporting the data grid to Excel with sorting and filtering options enabled on the column header in Excel sheet by setting the `DataGridExcelExportingOption.CanAllowSortingAndFiltering` to `true`. The default value of this property is false.
```

Comment:
The SfDataGrid enables the export of the data grid to Excel while allowing sorting and filtering options to be enabled on the column header in the Excel sheet. This can be achieved by setting the `DataGridExcelExportingOption.CanAllowSortingAndFiltering` property to `true`. By default, this property is set to false.

Reply:
updated